src/jit/lsra.cpp

```diff
@@ -4169,7 +4231,9 @@ BasicBlock* getNonEmptyBlock(BasicBlock* block)
 // insertZeroInitRefPositions: Handle lclVars that are live-in to the first block
 //
 // Notes:
-//    For each lclVar that is live-in to the first block:
+//    Prior to calling this method, 'currentLiveVars' must be set to the set of register
```

Comment:
If this is simply to eliminate a redundant bit set intersection, I'd love to see an assert that verifies the contents of `currentLiveVars`.